layouts/fastly/helix.vcl

```diff
@@ -801,7 +801,6 @@ sub hlx_fetch_static {
     set req.http.X-Trace = req.http.X-Trace + "(400)";
     # Cache for a short time, restart will get rid of it anyway
     set beresp.ttl = 60s;
-    return(deliver);
```

Comment:
@trieloff That's the line that causes 404 to not use the default error page.
I tested cases with (404.html at root of the project) and without the custom error page, it renders the expect one.
I just wonder if this does not break something else.

Reply:
Even if it does, the scope of breaking something else should be quite small.